src/main/java/edu/unc/lib/boxc/migration/cdm/CdmIndexCommand.java

```diff
@@ -47,32 +53,54 @@ public Integer call() throws Exception {
         try {
             initialize();
 
-            // if user provides csv, check that it exists
+            if (options.getCsvFile() != null && options.getEadTsvFile() != null) {
+                throw new IllegalArgumentException("CSVs and EAD to CDM TSVs may not be used in the same indexing command");
+            }
+
+            // if user provides a file, check that it exists
             if (options.getCsvFile() != null) {
                 if (Files.exists(options.getCsvFile())) {
-                    CdmFieldInfo csvExportFields = fieldService.retrieveFieldsFromCsv(options.getCsvFile());
+                    CdmFieldInfo csvExportFields = fieldService.retrieveFields(options.getCsvFile(), CSV);
                     fieldService.persistFieldsToProject(project, csvExportFields);
+                    indexService.createDatabase(fieldService, project, options);
+                    fileIndexService.setSource(CSV);
+                    fileIndexService.indexAllFromFile(options);
                 } else {
                     throw new MigrationException("No csv file exists in " + options.getCsvFile());
                 }
-            }
-
-            indexService.createDatabase(options);
-            indexService.index(options);
-            // Display any warning messages to user
-            if (!indexService.getIndexingWarnings().isEmpty()) {
-                indexService.getIndexingWarnings().forEach(msg -> outputLogger.info(msg));
+            } else if (options.getEadTsvFile() != null) { // if user provides EAD to CDM tsv, check that it exists
+                if (Files.exists(options.getEadTsvFile())) {
+                    // standardize headers and add CDM ID column
+                    var formattedEadToCdmTsvPath = fileIndexService.addIdsToEadToCdmTsv(options.getEadTsvFile());
+                    CdmFieldInfo csvExportFields = fieldService.retrieveFields(formattedEadToCdmTsvPath, CdmFieldService.EAD_TO_CDM);
```

Comment:
```suggestion
                    CdmFieldInfo csvExportFields = fieldService.retrieveFields(formattedEadToCdmTsvPath, EAD_TO_CDM);
```
For consistency, or do the opposite and prefix these constants everywhere